lib/Db/FaceMapper.php

```diff
@@ -38,7 +38,7 @@ public function __construct(IDBConnection $db) {
 
 	public function find (int $faceId) {
 		$qb = $this->db->getQueryBuilder();
-		$qb->select('id', 'image', 'person', 'left', 'right', 'top', 'bottom', 'descriptor')
+		$qb->select('id', 'image', 'person', 'left', 'right', 'top', 'bottom', 'landmarks', 'descriptor', 'confidence')
```

Comment:
#consider making landmarks under boolean argument (but I am not even sure we need this micro-optimization), feel free to ignore this:D

Reply:
I think I'm not going to do it, because it is a very punctual call, requesting a particular face, and I understand that it must return the entire entity.
If it is necessary in other calls surely I will do it, but 'find()' should only search by id without any restriction.